src/modules/role.js

```diff
@@ -0,0 +1,40 @@
+const Eris = require("eris");
+const config = require("../config");
+const utils = require("../utils");
+const threadUtils = require("../threadUtils");
+
+/**
+ * @param {Eris.CommandClient} bot
+ */
+module.exports = bot => {
+  threadUtils.addInboxServerCommand(bot, "role", async (msg, args, thread) => {
+    if (! thread) return;
+    if (! args[0] || ! config.inboxAdminRoleId || (! msg.member.roles.includes(config.inboxAdminRoleId) && ! msg.member.roles.includes("523021576128692239"))) {
```

Comment:
I feel like this massive condition could be cleaned up

Reply:
Updated, lmk if that's better.